index.html

```diff
@@ -0,0 +1,474 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src='main.js'></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body onload="modalDisplay()">
+    <div class="nav_modal-background">
+      <div class="nav_modal">
+        <div class="nav_modal-header">
+          <h2 class="nav_modal-title"></h2>
+          <i class="fa fa-times service-icon X"></i>
+          <!-- <div class="X">
+            <p>X</p>
+          </div> -->
+        </div>
+        <div class="nav_modal-body">
+            <ul class="nav_ul">
+              <li class="nav_li">
+                <a class=" modal_nav_link " href="#intro">Home</a>
+              </li>
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#projects">Portfolio</a>
+              </li>
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#about-me">About me</a>
+              </li>
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#contact_me">Contact me</a>
```

Comment:
- [ ] Please change the value of the href from contact_me to contact to make it work